snake.js

```diff
@@ -287,9 +284,15 @@
       stopMotion({horizontal:false,length: -dy*4});
       y-=dy*5;
     }
-
+    if(lives === 0){
+      gameOver();

```

Comment:
Any specific reason for putting this inside `updateHead` ? 
I think the purpose of updateHead is to only update the positions of head & not anything else, It'll make it easier to track bugs later on & the code will be easier to understand.

The code related to game state should be belong in the main game loop i.e `draw()` function which is the main game loop.